cps/static/js/table.js

```diff
@@ -93,6 +93,116 @@ $(function() {
         var domainId = $(e.relatedTarget).data("domain-id");
         $(e.currentTarget).find("#btndeletedomain").data("domainId", domainId);
     });
+
+    $('#restrictModal').on('hidden.bs.modal', function () {
+        // Destroy table and remove hooks for buttons
+        $("#restrict-elements-table").unbind();
+        $('#restrict-elements-table').bootstrapTable('destroy');
+        $("[id^=submit_]").unbind();
+        $('#h1').addClass('hidden');
+        $('#h2').addClass('hidden');
+        $('#h3').addClass('hidden');
+        $('#h4').addClass('hidden');
+    });
+    function startTable(type){
+        var pathname = document.getElementsByTagName("script"), src = pathname[pathname.length-1].src;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Operator '-' must be spaced.](https://app.codacy.com/manual/Ozzie-IsaacsOrganization/calibre-web/pullRequest?prid=4846308)